taskcluster/docker/linux/Dockerfile

```diff
@@ -29,7 +29,10 @@ ENV SHELL=/bin/bash \
     HOME=/builds/worker \
     PATH=/builds/worker/.local/bin:/root/.local/bin:$PATH
 
-ENV MOZ_LOG=HTMLMediaElement:4,HTMLMediaElementEvents:4,cubeb:5,PlatformDecoderModule:5,AudioSink:5,AudioSinkWrapper:5,MediaDecoderStateMachine:4,MediaDecoder:4,MediaFormatReader:5,GMP:5,EME:5
+# Uncomment to get process logs
```

Comment:
its ok to have env variables right? (like commited)

Reply:
Yeah, it's ok, but this one got removed / commented because it's going to produce artifact trash in our CI runs.